.github/workflows/ClangFormat.yml

```diff
@@ -15,6 +15,6 @@ jobs:
           install-llvm: true # Needed to configure jlm
           install-clang-format: true
       - name: "Configure jlm with HLS and MLIR enabled"
-        run: ./configure.sh --enable-mlir --enable-hls
+        run: ./configure.sh --enable-mlir /usr --enable-hls /usr
```

Comment:
Could we add a comment here that explains why we hand in a dummy folder, please? Future us/me won't remember when I look at this.